assignment_8.py

```diff
@@ -0,0 +1,6 @@
+list_1 = [144, 23, 53, 64, 76, 85, 342, 56, 23]
+
+list_1.sort()
+print(f'second largest number is :- ', list_1[-2])

```

Comment:
If you are using f string then you can use curly braces inside string. like :
print(f'second largest number is :- {list_1[-2]}')